### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
added cart page.

### CODE REVIEW

**Review by teja538 — APPROVED** [review]

**Review by Sharmaryan — APPROVED** [review]

**Comment by SuchitaGhadge** [comment]
On navigation , login in button should not be there in cart page.  Cart is visible after a successful sign in only. Design looks good.

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    </header>
+
+    <!-- MAIN CONTAINER -->
+    <main class="main-container">
+        <h1 class="align-center">My Cart</h1>
+
+        <div class="cart-full two-item-container ratio-70-30">
+            <section class="cart-section">
+                <div class="ecom-card-h br-sm">
+                    <img class="card-img br-sm img-responsive h-card-img" src="https://picsum.photos/150/150" alt=""/>
```
Comment (rishichourasia) [inline pages/cart.html]: Empty alt tag . Provide one

**Review by rishichourasia — COMMENTED** [review]
Few UI suggestion i would like to make - 

```
1 - Background color looks inconsistent through the pages . footer has different colour and sectiion has different . try to keep them consistent .
2- Card width and height looks absurdly big compared to billing info box . maybe reduce that to around 25rems. 
3- Does not shrink on responsiveness even tho you have written code for it . 
4 - Overall UI can be improved with minor changes , look into other ecom for ideas.

**Comment by muskan-gupta-0803** [comment]
> On navigation , login in button should not be there in cart page. Cart is visible after a successful sign in only. Design looks good.

oh yes, I will remove it.

File: pages/cart.html (hunk 1 above)
Comment (muskan-gupta-0803) [reply pages/cart.html]: added

**Comment by muskan-gupta-0803** [comment]
> Few UI suggestion i would like to make -
> 
> ```
> 1 - Background color looks inconsistent through the pages . footer has different colour and sectiion has different . try to keep them consistent .
> 2- Card width and height looks absurdly big compared to billing info box . maybe reduce that to around 25rems. 
> 3- Does not shrink on responsiveness even tho you have written code for it . 
> 4 - Overall UI can be improved with minor changes , look into other ecom for ideas.
> ```

Sure i will look into this.

**Pull request merged**